app/window.go

```diff
@@ -525,4 +526,20 @@ func MinSize(w, h unit.Value) Option {
 	}
 }
 
+// StatusBarColor sets the color of the status bar (which usually contains notification icons
+// and system icons).
```

Comment:
Be specific:

// StatusBarColor sets the color of the Android status bar.

Consider shortening to "StatusColor".